### PR TITLE
Feature/external sensors svs

### PR DESCRIPTION
**Pull request by rafaelbarretorb**
### Features Description
- Add new ROS custom message for external sensors ```SVS.msg```
- Publish message from the external sensor SVS

### Notes
 - New feature tested using the INS Rovins Nano and Valeport miniSVS.

### CODE REVIEW

**Comment by rafaelbarretorb** [comment]
Message in the terminal:

image

File: ixblue_ins_driver/src/ros_publisher.cpp [hunk 1]
```diff
@@ -461,3 +470,19 @@ ROSPublisher::toiXInsMsg(const ixblue_stdbin_decoder::Data::BinaryNav& navData)
...
 }
+
+ixblue_ins_msgs::ExternalSensorDataPtr
+ROSPublisher::toiXExternalSensorMsg(const ixblue_stdbin_decoder::Data::BinaryNav& navData) {
+  // --- Check if there are enough data to send the message
+  if(navData.soundVelocity.is_initialized() == false) {
+      return nullptr;
+  }
+
+  // --- Initialisation
```
Comment (carlospereira179) [inline ixblue_ins_driver/src/ros_publisher.cpp:481]: Initialization

**Review by carlospereira179 — APPROVED** [review]

**Review by caioaamaral — APPROVED** [review]
LGTM, I would just suggest renaming `ExternalSensor` prefix to `SVS`

File: ixblue_ins_driver/src/ros_publisher.cpp (hunk 1 above)
Comment (rafaelbarretorb) [reply ixblue_ins_driver/src/ros_publisher.cpp:481]: It is not incorrect and I prefer with with z too. It is a UK english and ixblue guys use this way in other parts of the code.

File: ixblue_ins_msgs/msg/ExternalSensorData.msg [hunk 2]
```diff
@@ -0,0 +1,5 @@
+Header header # Standard ROS message header
+
+# Sound Velocity Sensor (SVS)
+int32 validity_time  # Data validity time Time tag in steps of 100 µs
+float32 sound_velocity  # in meter/second
```
Comment (caioaamaral) [inline ixblue_ins_msgs/msg/ExternalSensorData.msg]: We could call this just `SVS.msg`

**Comment by rafaelbarretorb** [comment]
> LGTM, I would just suggest renaming `ExternalSensor` prefix to `SVS`

I thought using the same message for all external sensors, and `ExternalSensors` would be better. But now I think using two different messages names (SVS and DVL). In the next PR I will decide.

File: ixblue_ins_msgs/msg/ExternalSensorData.msg (hunk 2 above)
Comment (rafaelbarretorb) [reply ixblue_ins_msgs/msg/ExternalSensorData.msg]: I decided to change it.

**Pull request merged**